pkg/storage/backend-storage/backend-storage.go

```diff
@@ -499,6 +503,15 @@ func (bs *BackendStorage) createPVC(vmi *corev1.VirtualMachineInstance, labels m
 			*metav1.NewControllerRef(vmi, corev1.VirtualMachineInstanceGroupVersionKind),
 		}
 	}
+
+	// Adding this label to allow the PVC to be processed by the CDI WebhookPvcRendering mutating webhook,
+	// which must be enabled in the CDI CR via feature gate.
+	// This mutating webhook processes the PVC based on its associated StorageProfile.
+	// For example, a profile can define a minimum supported volume size via the annotation:
+	// cdi.kubevirt.io/minimumSupportedPvcSize: 4Gi
+	// This helps avoid issues with provisioners that reject the hardcoded 10Mi PVC size used in KubeVirt.
+	labels[LabelApplyStorageProfile] = "true"
```

Comment:
probably want to unit test this to prevent us from regressing (piggy back on existing tests even)

Reply:
ack